packages/trader/src/Stores/Modules/Trading/trade-store.ts

```diff
@@ -1502,6 +1534,21 @@ export default class TradeStore extends BaseStore {
         });
     }
 
+    setChartModeFromURL() {
+        const { chartType, granularity: newGranularity } = getTradeURLParams();
```

Comment:
Maybe it'll make sense to rename `chartType: newChartType` as you did for `granularity`? The reason is that when all namings will be in camelCase we'll get 2 constants with same name.
`chart_type `(from  `this.root_store.contract_trade`) -----> `chartType`
`chartType` (from `getTradeURLParams()`) -----> also `chartType`
It's absolutely optional, you can ignore if not relevant)

Reply:
good idea, i've found better names for them👍🏻